WalletWasabi.Fluent/ViewModels/Dialogs/CancelTransactionDialogViewModel.cs

```diff
@@ -0,0 +1,81 @@
+using System.Linq;
+using System.Reactive;
+using System.Threading.Tasks;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Blockchain.TransactionBuilding;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Logging;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Dialogs;
```

Comment:
Move it to a proper place, like `History/Features`

Reply:
Moved under **ViewModels\Wallets\Home\History\Features**